### PR TITLE
ceph-disk: fix possible bugs

### PR DESCRIPTION
**Pull request by shun-s**
fix possible bugs

Signed-off-by: Song Shun <song.shun3@zte.com.cn>

### CODE REVIEW

**Comment by tchaikov** [comment]
@shun-s could you be more specific in the title?

**Comment by shun-s** [comment]
@tchaikov  update the tittle, how about that ?

File: src/ceph-disk/ceph_disk/main.py [hunk 1]
```diff
@@ -1852,8 +1852,10 @@ def ptype_for_name(self, name):
     @staticmethod
     def factory(path, dev, args):
         dmcrypt_type = CryptHelpers.get_dmcrypt_type(args)
-        if ((path is not None and is_mpath(path)) or
-                (dev is not None and is_mpath(dev))):
+        if path is None and dev is None:
+            raise Error("device is None")
+
+        if is_mpath(path) or is_mpath(dev):
```
Comment (tchaikov) [inline src/ceph-disk/ceph_disk/main.py]: what if path is None or dev is None?

**Comment by tchaikov** [comment]
please reopen this PR if you believe there is a valid case for your fix.

**Pull request closed without merging**